doc/manual/src/language/values.md

```diff
@@ -14,8 +14,11 @@
   respectively.
 
   You can include the result of an expression into a string by
-  enclosing it in `${...}`, a feature known as *antiquotation*. The
-  enclosed expression must evaluate to something that can be coerced
+  enclosing it in `${...}`, a feature known as *string interpolation*.
+  Such a string with placeholders is an *interpolated string*, and a
```

Comment:
I think we shouldn't use "placeholders", because that sounds like Python/Rust/SQL placeholders (e.g. `{}`` or `format!("foo {}", x)`). Nix doesn't have placeholders in that sense.

Reply:
```suggestion
  Such a string is an *interpolated string*, and a
```